draft-ietf-quic-transport.md

```diff
@@ -1544,7 +1588,6 @@ that are smaller than the remembered value of the parameters.
 * initial_max_stream_data_uni
 * initial_max_streams_bidi
 * initial_max_streams_uni
-* active_connection_id_limit
```

Comment:
Why was this removed in this PR?

Reply:
A rebasing error.  I added something to this list, sorted it, removed it, now fixed it.